### ISSUE TITLE
Steam

### ISSUE THREAD

**Issue by vexual-sy** [endless-sky/endless-sky#445]
Hey Mates :)

I just want to first of all thank you all for getting this game into my Hands, seriously, i have like 50 hours played.
What id like to bring to you attention, is Steam.
I know the game is in steam, but an update on what is happening with the game ie patch and versions 1.1+ info, would be great to see.
many people want to see this game progress into v1.1 i would LOVE to see if become atleast completed.

I would also put a price on it, nothing huge, but if you're going to be spending serious time, making the game all that it should be, we should be able to atleast pay you.

so in a nutshell, more steam updates, or more information updates into steam would be rad.

**Comment by endless-sky** [comment]
New releases get uploaded to Steam the same day they become available here. For the first week I make them available only to people who have opted into the "beta" branch, and if no major bugs are found I make the new release the default.

There's a [development roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap) available on the wiki.

I work as a salaried software engineer and, as is common for job contracts in technology fields, my contract doesn't let me have a second paid job at the same time. So, I'm treating the game development as something I'm doing for free as a hobby.
